relayer/relays/beacon/config.go

```diff
@@ -21,8 +31,18 @@ type ContractsConfig struct {
 type BeaconConfig struct {
 	Endpoint                string `mapstructure:"endpoint"`
 	FinalizedUpdateEndpoint string `mapstructure:"finalizedUpdateEndpoint"`
+	Spec                    Spec   `mapstructure:"spec"`
+	ActiveSpec              string `mapstructure:"activeSpec"`
 }
 
 type SinkConfig struct {
 	Parachain config.ParachainConfig `mapstructure:"parachain"`
 }
+
+func (c Config) GetSpecSettings() SpecSettings {
+	if c.Source.Beacon.ActiveSpec == "minimal" {
```

Comment:
It would be cool if you could some how query the beacon spec from the parachain or beacon.

Reply:
Totally! I've logged issue [SNO-296](https://linear.app/snowfork/issue/SNO-296/query-parachain-feature-for-beacon-spec) for it.